vllm/v1/engine/detokenizer.py

```diff
@@ -189,85 +178,3 @@ def _get_next_output_text(self, finished: bool, delta: bool) -> str:
             self._last_output_text_offset = length
             return self.output_text[last_offset:length]
         return ""
-
-
-class Detokenizer:
```

Comment:
This is now called `OutputProcessor`, since we will do more than "just" detokenization in the `step()` function